packages/wdio-cli/src/constants.js

```diff
@@ -18,6 +18,18 @@ To run your tests, execute:
 $ wdio run wdio.conf.js
 `
 
+export const ANDROID_CONFIG = {
+    platformName: 'Android',
+    automationName: 'UiAutomator2',
+    deviceName: 'Test'
+}
+
+export const IOS_CONFIG = {
+    platformName: 'iOS',
+    automationName: 'XCUITest',
+    deviceName: 'iPhone 8'
```

Comment:
is there a simple way to get available deviceNames without adding weird dependencies?

Reply:
We can replace it with 'iPhone Simulator' then we don't need any additional capabilities.
Appium will get first available simulator in list.
I don't know why I haven't seen this solution earlier.

Should we keep options to be able to configure udid/platformVersion/deviceName?

Reply:
yeah, in case someone wants to run a repl on his connected real device